tests/parquet/data/export_info_ci.json

```diff
@@ -0,0 +1 @@
+{"exportTaskIdentifier":"ci","sourceArn":"arn:aws:rds:us-west-2:697156367097:cluster-snapshot:ci","exportOnly":[],"snapshotTime":"Sep 10, 2020 6:50:05 AM","taskStartTime":"Sep 10, 2020 7:12:21 AM","taskEndTime":"Sep 10, 2020 7:15:14 AM","s3Bucket":"handlerww-cluster","s3Prefix":"","exportedFilesPath":"ci","iamRoleArn":"arn:aws:iam::697156367097:role/service-role/export-to-s3","kmsKeyId":"arn:aws:kms:us-west-2:697156367097:key/9fd6e56f-de78-4229-9044-3307c4836bc5","status":"COMPLETE","percentProgress":0,"totalExportedDataInGB":1.0}
```

Comment:
do we need to include these metadata files 🤔

Reply:
Not really, they are useless actually. But since in the user case, these meta files are also exist, I think there is no harm to include them in the test.

Reply:
ok.